### ISSUE TITLE
Docs: Icon Only Button Canvas demo is missing aria-label

### ISSUE THREAD

**Issue by coliff** [trimble-oss/modus-web-components#2652]
Link: https://modus-web-components.trimble.com/?path=/story/components-button--icon-only

![image](https://github.com/trimble-oss/modus-web-components/assets/1212885/3423b31c-5f01-409d-876d-edd5a2255ea3)
